main.js

```diff
@@ -85,11 +89,120 @@ function cardsBuilder(card, index) {
+    skills: ['html', 'css', 'javascript', 'ruby', 'github', 'Bootstraps'],
+    image: 'images/snapshot5.png',
+    liveLink: 'See live',
+    sourceLink: 'See Source',
+  },
+  {
+    title: 'Multi-Post Stories',
+    datas: {
+      author: 'FACEBOOK',
+      role: 'Full Stack Dev',
+      year: '2015',
+    },
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's .......",
+    skills: ['html', 'css', 'javascript', 'ruby', 'github', 'Bootstraps'],
+    image: 'images/snapshot4.png',
+    liveLink: 'See live',
+    sourceLink: 'See Source',
+  },
+  {
+    title: 'Facebook 360',
+    datas: {
+      author: 'FACEBOOK',
+      role: 'Full Stack Dev',
+      year: '2015',
+    },
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's .......",
+    skills: ['html', 'css', 'javascript', 'ruby', 'github', 'Bootstraps'],
+    image: 'images/snapshot1.png',
+    liveLink: 'See live',
+    sourceLink: 'See Source',
+  },
+  {
+    title: 'Uber Navigation',
+    datas: {
+      author: 'Uber',
+      role: 'Lead Developer',
+      year: '2018',
+    },
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's...........",
+    skills: ['html', 'css', 'javascript', 'ruby', 'github', 'Bootstraps'],
+    image: 'images/snapshot2.png',
+    liveLink: 'See live',
+    sourceLink: 'See Source',
+  },
+];
+
+const btns = document.querySelectorAll('.btn');
+const bgModal = document.querySelector('.bg-modal');
+function popUp(index) {
+  const {
+    title, datas, description, skills, image,
+  } = popupCards[index];
+  const skillsHtml = skills.map((s) => `<li class="tag html-tag">${s}</li>`).join('');
+  bgModal.innerHTML = `
+  <div class = "modal-content">
+        <div class="modal-header">
+          <h2 class="title">${title}</h3>
+          <span class="close-button">+</span>
+        </div>
+        <ul class = "frame frame-${(index === 0 || index % 2 === 0) ? '1' : '2'}">
+          <li class = "list-item author">${datas.author}</li>
+          <li class = "list-item role">${datas.role}</li>
+          <li class = "list-item year">${datas.year}</li>
+        </ul>
+        <img src="${image}" alt="first-project" class="snapshotdesktop1"/>
+        <div class = "btm-content">
+          <div class = "descr-1">
+            <p class="primary-text2">
+            ${description}
+            </p>
+          </div>
+          <div>
+            <ul class="tags">
+              ${skillsHtml}
+            </ul>
+            <div class="buttons">
+              <button type ="button" class="live-btn"> See Live <i class="fas fa-external-link-alt"></i></button>
+              <button type ="button" class="live-btn"> See Source <i class="fab fa-github"></i></button>
+            </div>
+          </div>
+        </div>
+  </div>
```

Comment:
- Good job for the work done so far 👍 👍 . There are some little issues with your project.  When I scroll, other sections of your project overlap on your popup modal. Please reference the picture attached below 👇, I advise you apply the `z-index` property on the modal. You may read this [article](https://www.w3schools.com/cssref/pr_pos_z-index.php) for more info

Your project:
<img width="550" alt="ffffb" src="https://user-images.githubusercontent.com/73607512/202445572-34d7b26e-7050-4311-9112-1cacaf262c0b.PNG">